python/scope.py

```diff
@@ -798,7 +799,7 @@ def eval_value(self, value):
                 return float(value)
             return int(value)
         elif value.type == "STRING":
-            return bytes(unescape_sequence(value[1:-1]), "utf-8").decode("unicode_escape")
+            return bytes(unescape(value[1:-1]), "utf-8").decode("unicode_escape")
```

Comment:
Is `bytes(..., "utf-8").decode("unicode_escape")` necessary?

```suggestion
            return bytes(unescape(value[1:-1]), "utf-8").decode("unicode_escape")
```